src/otx/core/data/transform_libs/utils.py

```diff
@@ -104,14 +104,29 @@ def __get__(self, obj, cls):  # noqa: ANN001
         return copy.copy(self)
 
 
-def to_np_image(img: Tensor) -> np.ndarray:
+def get_image_shape(img: np.ndarray | Tensor | list) -> tuple[int, int]:
+    """Get image(s) shape with (height, width)."""
+    if isinstance(img, np.ndarray):
+        return img.shape[:2]
+    if isinstance(img, Tensor):
+        return img.shape[-2:]
+    if isinstance(img, list):
+        return get_image_shape(img[0])
+
+    msg = f"{type(img)} is not supported."
+    raise TypeError(msg)
```

Comment:
According to the type hint, a type error should be unreachable. Could you add `isinstance(img, (np.ndarray, Tensor, list))` at the start of the function to ensure type checking?

Reply:
Added https://github.com/openvinotoolkit/training_extensions/pull/3644/commits/896cfe473a106ab0c3216fe24aaebd41203974ad